### ISSUE TITLE
[Bug]: panic: invalid channel name: milvus-rootcoord-dml_0 when upgrade 2.0.2 standalone to master

### ISSUE THREAD

**Issue by ThreadDao** [milvus-io/milvus#17329]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20220524-d05c24a0
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

1. Deploy standalone `2.0.2`
2. Run test_e2e.py without dropping the collection
3. Upgrade release to master image `master-20220524-d05c24a0`
```
[2022/06/01 13:53:40.540 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-delta_253] [elapsed=0]
[2022/06/01 13:53:40.540 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-delta_254] [elapsed=0]
[2022/06/01 13:53:40.540 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-delta_255] [elapsed=0]
[2022/06/01 13:53:40.540 +00:00] [DEBUG] [dml_channels.go:72] ["init dml channels"] [num=256]
[2022/06/01 13:53:40.540 +00:00] [ERROR] [dml_channels.go:157] ["invalid channel name"] [chanName=milvus-upg-rootcoord-dml_0] [stack="github.com/milvus-io/milvus/internal/rootcoord.(*dmlChannels).addChannels\n\t/go/src/github.com/milvus-io/milvus/internal/rootcoord/dml_channels.go:157\ngithub.com/milvus-io/milvus/internal/rootcoord.newTimeTickSync\n\t/go/src/github.com/milvus-io/milvus/internal/rootcoord/timeticksync.go:97\ngithub.com/milvus-io/milvus/internal/rootcoord.(*Core).Init.func3\n\t/go/src/github.com/milvus-io/milvus/internal/rootcoord/root_coord.go:1122\nsync.(*Once).doSlow\n\t/usr/local/go/src/sync/once.go:68\nsync.(*Once).Do\n\t/usr/local/go/src/sync/once.go:59\ngithub.com/milvus-io/milvus/internal/rootcoord.(*Core).Init\n\t/go/src/github.com/milvus-io/milvus/internal/rootcoord/root_coord.go:1052\ngithub.com/milvus-io/milvus/internal/distributed/rootcoord.(*Server).init\n\t/go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/service.go:220\ngithub.com/milvus-io/milvus/internal/distributed/rootcoord.(*Server).Run\n\t/go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/service.go:139\ngithub.com/milvus-io/milvus/cmd/components.(*RootCoord).Run\n\t/go/src/github.com/milvus-io/milvus/cmd/components/root_coord.go:55\ngithub.com/milvus-io/milvus/cmd/roles.(*MilvusRoles).runRootCoord.func1\n\t/go/src/github.com/milvus-io/milvus/cmd/roles/roles.go:114"]
panic: invalid channel name: milvus-upg-rootcoord-dml_0

goroutine 135 [running]:
github.com/milvus-io/milvus/internal/rootcoord.(*dmlChannels).addChannels(0xc0016b00e0, 0xc000bf0480, 0x2, 0x2)
    /go/src/github.com/milvus-io/milvus/internal/rootcoord/dml_channels.go:158 +0x30d
github.com/milvus-io/milvus/internal/rootcoord.newTimeTickSync(0x4271ef0, 0xc000c36fc0, 0x28, 0x7f998c244708, 0xc000c3f290, 0xc001788870, 0x0)
    /go/src/github.com/milvus-io/milvus/internal/rootcoord/timeticksync.go:97 +0x459
github.com/milvus-io/milvus/internal/rootcoord.(*Core).Init.func3()
    /go/src/github.com/milvus-io/milvus/internal/rootcoord/root_coord.go:1122 +0xa79
sync.(*Once).doSlow(0xc00079e300, 0xc000ef9c68)
    /usr/local/go/src/sync/once.go:68 +0xec
sync.(*Once).Do(...)
    /usr/local/go/src/sync/once.go:59
github.com/milvus-io/milvus/internal/rootcoord.(*Core).Init(0xc00079e1a0, 0x429fe70, 0xc000f36090)
    /go/src/github.com/milvus-io/milvus/internal/rootcoord/root_coord.go:1052 +0x2ac
github.com/milvus-io/milvus/internal/distributed/rootcoord.(*Server).init(0xc000c68000, 0x8bdf3e, 0xc0007e8648)
    /go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/service.go:220 +0x612
github.com/milvus-io/milvus/internal/distributed/rootcoord.(*Server).Run(0xc000c68000, 0xc000c68000, 0x0)
    /go/src/github.com/milvus-io/milvus/internal/distributed/rootcoord/service.go:139 +0x2f
github.com/milvus-io/milvus/cmd/components.(*RootCoord).Run(0xc000c37000, 0xffffffffffffffff, 0x42802b8)
    /go/src/github.com/milvus-io/milvus/cmd/components/root_coord.go:55 +0x33
github.com/milvus-io/milvus/cmd/roles.(*MilvusRoles).runRootCoord.func1(0xc000b3c501, 0x4271ef0, 0xc0005e3e80, 0xc00078a058, 0xc0007e8640)
    /go/src/github.com/milvus-io/milvus/cmd/roles/roles.go:114 +0x125
created by github.com/milvus-io/milvus/cmd/roles.(*MilvusRoles).runRootCoord
    /go/src/github.com/milvus-io/milvus/cmd/roles/roles.go:97 +0xb0
```
[standalone.log](https://github.com/milvus-io/milvus/files/8814785/standalone.log)

**Comment by cydrain** [comment]
consult with @ThreadDao , this issue can reproduce manually, but cannot reproduce using git action automatic scripts.
Need double check whether it's a real issue.

**Comment by xiaofan-luan** [comment]
milvus-upg  and by-dev seems to be two different cluster prefix~

**Comment by cydrain** [comment]
@ThreadDao 
From the log
```
[2022/06/01 13:53:40.540 +00:00] [ERROR] [dml_channels.go:157] ["invalid channel name"] [chanName=milvus-upg-rootcoord-dml_0] 
```
we can see the cluster prefix is "milvus-upg", by default cluster prefix is set to "by-dev".
Is this the root cause that makes upgrade fail ?

**Comment by ThreadDao** [comment]
> @ThreadDao From the log
> 
> ```
> [2022/06/01 13:53:40.540 +00:00] [ERROR] [dml_channels.go:157] ["invalid channel name"] [chanName=milvus-upg-rootcoord-dml_0] 
> ```
> 
> we can see the cluster prefix is "milvus-upg", by default cluster prefix is set to "by-dev". Is this the root cause that makes upgrade fail ?

'milvus-upg' is my standalone release name

**Comment by haorenfsa** [comment]
> @ThreadDao From the log
> 
> ```
> [2022/06/01 13:53:40.540 +00:00] [ERROR] [dml_channels.go:157] ["invalid channel name"] [chanName=milvus-upg-rootcoord-dml_0] 
> ```
> 
> we can see the cluster prefix is "milvus-upg", by default cluster prefix is set to "by-dev". Is this the root cause that makes upgrade fail ?

milvus-operator always uses `${milvus-name}-` as its channelPrefix by default.
So the channelPrefix should be `${milvus-name}-` both before & after the upgrade.

**Comment by cydrain** [comment]
```
[2022/06/01 13:53:40.520 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-dml_0] [elapsed=0]
[2022/06/01 13:53:40.521 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-dml_1] [elapsed=0]
[2022/06/01 13:53:40.521 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-dml_2] [elapsed=0]
[2022/06/01 13:53:40.521 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-dml_3] [elapsed=0]
[2022/06/01 13:53:40.521 +00:00] [DEBUG] [rocksmq_impl.go:328] ["Rocksmq create topic successfully "] [topic=by-dev-rootcoord-dml_4] [elapsed=0]
... ...
```
see above logs, when new version Milvus startup, cluster name "by-dev" is used, not "${milvus-name}"

**Comment by cydrain** [comment]
@haorenfsa 
Root cause found.
In Milvus 2.0.2, cluster prefix is named "msgChannel.chanNamePrefix.cluster".
In Milvus latest, cluster prefix is re-named to "common.chanNamePrefix.cluster".
For backward compatibility, Milvus will get cluster prefix from "common.chanNamePrefix.cluster" firstly, then from "msgChannel.chanNamePrefix.cluster".
The latest k8s Operator only change the name in "msgChannel.chanNamePrefix.cluster" but not in "common.chanNamePrefix.cluster".

**Comment by ThreadDao** [comment]
Fixed `master-20220608-6d078ebe`
